src/TiciTacaToeyGameEngine.ts

```diff
@@ -207,6 +208,10 @@ class TiciTacaToeyGameEngine implements GameEngine {
         break;
       }
       case MessageTypes.JOIN_GAME: {
+        if (!this.players[message.playerId]){
```

Comment:
Can we not use the same check as above to check for existence of a property in object ?

if (!this.players[message.playerId]){

Reply:
oh yes that's my bad, updating now!